cinch/roles/check_ssh/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+ansible_port: 22
+ssh_retries: 100
+ssh_retry_delay: 6
+host: "{{ hostvars[inventory_hostname]['ansible_host'] | default(inventory_hostname) }}"
```

Comment:
hostvars[inventory_hostname] is probably redundant here. By default, values like "ansible_host" are pulled from the current host's variables. I don't object to being explicit, here, as this is intentionally kludgy code to work around Ansible's internal aversion to recovery from SSH host unreachable. I just wanted you to be aware of that.